packages/react-aria-components/stories/index.stories.tsx

```diff
@@ -674,3 +676,37 @@ function CustomTab(props) {
       })} />
   );
 }
+
+function Draggable() {
+  let {dragProps, isDragging} = useDrag({
+    getItems() {
+      return [{
+        'text/plain': 'hello world',
+        'my-app-custom-type': JSON.stringify({message: 'hello world'})
+      }];
+    }
+  });
+
+  return (
+    <div
+      {...dragProps}
+      role="button"
+      tabIndex={0}
+      className={`draggable ${isDragging ? 'dragging' : ''}`}>
+      Drag me
+    </div>
+  );
+}
+
+export const DropZoneExample = () => (
```

Comment:
Can we also get an example with render props? It could look something like this:

```tsx
    <DropZone 
      onDrop={action('On Drop')}
      onDropEnter={action('Drop Enter')}
      onDropExit={action('Drop Exit')}
      style={{border: '2px solid white', margin: '20px', padding: '20px'}}>
      {({isHovered, isFocused, isFocusVisible, isDropTarget}) => (
        <div>
          <div>DropZone</div>
          <div>isHovered: {isHovered ? 'true' : 'false'}</div>
          <div>isFocused: {isFocused ? 'true' : 'false'}</div>
          <div>isFocusVisible: {isFocusVisible ? 'true' : 'false'}</div>
          <div>isDropTarget: {isDropTarget ? 'true' : 'false'}</div>
        </div>
      )}
    </DropZone>
```